lib/synapse_pay_rest/api/subnets.rb

```diff
@@ -38,7 +38,8 @@ def get(user_id:, node_id:, subnet_id: nil, **options)
       end.compact
 
       path += '?' + params.join('&') if params.any?
-      tunnel = options[:full_dehydrate] == "yes"
+      tunnel = (options[:full_dehydrate].presence || options["full_dehydrate"].presence) == "yes"
```

Comment:
optional potential lint: ```tunnel = options.with_indifferent_access[:full_dehydrate] == "yes"```

Reply:
Thanks. That one would have been actually nice, but `with_indifferent_access` is added by Rails, over here it raises error :(